checker/manual/troubleshooting.tex

```diff
@@ -741,10 +736,30 @@ \subsection{Build the Checker Framework\label{building}}
 \end{Verbatim}
 % $ to unconfuse Emacs LaTeX mode
 
-\item Add \code{tools.jar} and \code{checker.jar} to your classpath.
-  (If you do not do this, you will have to supply the \code{-cp} option
-  whenever you run \code{javac} and use a checker plugin.)
-  Example command:
+Once \<checker.jar> is built, you may wish to put the Checker Framework \<javac> even earlier on
+your path. The Checker Framework's \code{javac} ensures that \<checker.jar> 
+is on your classpath and bootclasspath, but is otherwise
+identical to the Type Annotations compiler:
```

Comment:
This colon and following verbatim text is a bit confusing.  The verbatim text is related to the first sentence of the paragraph (it shows how to do that), but is not related to the second sentence that segues to it with a colon.